src/store/Controller.cc

```diff
@@ -829,7 +829,13 @@ Store::Controller::anchorToCache(StoreEntry &entry, bool &inSync)
 bool
 Store::Controller::smpAware() const
```

Comment:
If possible, please remove this non-static method by moving the smpAware() API from Storage to Disk (or something like that).